packages/theming/apple-theme/src/appleTypography.ios.ts

```diff
@@ -45,20 +49,91 @@ export function appleTypography(): Typography {
       heroSemibold: { face: 'primary', size: 'hero', weight: '600' },
       heroLargeStandard: { face: 'primary', size: 'heroLarge', weight: '400' },
       heroLargeSemibold: { face: 'primary', size: 'heroLarge', weight: '600' },
```

Comment:
Can we remove these older ones? Or does stuff still reference it?

Reply:
I'll have to defer to @rurikoaraki on this one.

Reply:
These are not optional properties on the Typography type. We're not removing the underlying types from the Typography yet because we'll probably break people if we do that right now. I think we need to get all the platforms onto the new tokens first, then we can set about trying to deprecate the old stuff.